Gateway/Request/ExpiryDateDataBuilder.php

```diff
@@ -40,13 +40,25 @@ public function __construct(
     public function build(array $buildSubject)
     {
         $paymentFormFields = $this->request->getParam('payment');
-        $expiryDate = date_create_from_format(
-            AdyenPayByLinkConfigProvider::DATE_TIME_FORMAT,
-            $paymentFormFields["adyen_pbl_expires_at"] . ' 23:59:59'
-        );
+        $expiryDate = null;
+        $payment = $buildSubject['payment']->getPayment()->getAdditionalInformation()['adyen_pbl_expires_at'];
```

Comment:
In `AdyenPayByLinkDataAssignObserver` there is a constant for this one. Should we move it in another location and used here too

Reply:
Thank you @AlexandrosMor, I changed to the const in all the locations